epub33/core/index.html

```diff
@@ -1492,21 +1486,24 @@ <h3>Data URLs</h3>
 					</li>
 				</ul>
 
+				<p>Although data URLs are considered part of their containing resource, because they have their own
+					media types they are still subject to the <a href="#sec-foreign-resources">foreign resource
+						restrictions</a>. EPUB creators MUST therefore encode data URLs as core media type resources or
+					use them where they can provide a fallback.</p>
```

Comment:
s/or use them where they can provide a fallback/or provide a fallback using intrinsic fallback mechanisms of the host format/

(reason: it is not because they can that they MUST 😉, and the suggested rewording is a bit more explicit).

We might add a note on the implicit consequence that data URLs cannot be used for non-CMT resources on elements that do not have intrinsic fallback capabilities, but I think it is clear enough already.